cli/src/config/templates.toml

```diff
@@ -171,7 +171,7 @@ if(commit.root(),
         if(commit.conflict(), label("conflict", "conflict")),
         if(config("ui.show-cryptographic-signatures").as_boolean(),
           format_short_cryptographic_signature(commit.signature())),
-        if(commit.empty(), label("empty", "(empty)")),
+        if(commit.empty(), empty_placeholder),
```

Comment:
since this isn't a placeholder, maybe it should be called "empty_commit_label" or something? ("label" is confusing in this context, but I don't have a better idea.)

Reply:
Oh! I kind of thought that's what it was...(since you used `placeholder` originally).

Reply:
I'll go with "empty_commit_marker". I think it's better than "label".